content/spin/cli-reference.md

```diff
@@ -410,13 +379,25 @@ SUBCOMMANDS:
 
 {{ blockEnd }}
 
+{{ startTab "v1.3"}}
+
+The `spin cloud` command is implemented by the [Fermyon Cloud Plugin](../cloud/cloud-command-reference).
+
+{{ blockEnd }}
+
 {{ blockEnd }}
 
-#### Deploy (Cloud)
+<!-- markdownlint-disable-next-line titlecase-rule -->
+## spin deploy
+
+`spin deploy` is a shotcurt to [`spin cloud deploy`](#cloud-deploy).
```

Comment:
```suggestion
`spin deploy` is a shortcut to [`spin cloud deploy`](#cloud-deploy).
```

Reply:
The link might be wrong too?  
```suggestion
`spin deploy` is a shortcut to [`spin cloud deploy`](#spin-cloud-deploy).
```

Though that just links on to the cloud command reference so maybe better to link directly from here to `/cloud/cloud-command-reference#spin-cloud-deploy`

Reply:
That would only be true for 1.3, so leaving this for older versions.